Shared/Scripts/bxl.ps1

```diff
@@ -239,7 +240,7 @@ if ($DeployStandaloneTest) {
 }
 
 if ($Vs) {
-    $AdditionalBuildXLArguments += "/p:[Sdk.BuildXL]GenerateVSSolution=true /vs";
+    $AdditionalBuildXLArguments += "/p:[Sdk.BuildXL]GenerateVSSolution=true /p:[Sdk.BuildXL]ExcludeBuildXLExplorer=1 /q:DebugNet472 /vs";

```

Comment:
not sure what `"/p:[Sdk.BuildXL]GenerateVSSolution=true` does

Reply:
excludes projects that are not buildable in VS